package.json

```diff
@@ -79,10 +79,10 @@
     "yo": "^3.1.1"
   },
   "dependencies": {
-    "@mapbox/geojson-extent": "^0.3.2",
-    "@mapbox/mapbox-gl-draw": "^1.2.0",
-    "@mapbox/mapbox-gl-geocoder": "^4.7.0",
+    "@mapbox/geojson-extent": "^1.0.1",
+    "@mapbox/mapbox-gl-draw": "^1.3.0",
+    "@mapbox/mapbox-gl-geocoder": "^4.7.4",
     "html-webpack-tags-plugin": "^2.0.17",
-    "mapbox-gl": "^1.12.0"
+    "mapbox-gl": "^2.3.1"
```

Comment:
Updates were necessary because they solve a couple of issues that were necessary for this feature